sources/packages/web/src/main.ts

```diff
@@ -5,8 +5,8 @@ import "primevue/resources/themes/md-light-indigo/theme.css";
 import "primevue/resources/primevue.min.css";
 import "primeicons/primeicons.css";
 import "primeflex/primeflex.css";
-
 import { createApp } from "vue";
+
```

Comment:
If this is the only change in the file, please consider removing it from the final commit.